cloud_images/centos7/dcos_vol_setup.sh

```diff
@@ -52,32 +52,8 @@ function main {
     fstab="$device $mount_location xfs defaults 0 2"
     echo "Adding entry to fstab: $fstab"
     echo "$fstab" >> /etc/fstab
-    if [ "$mount_location" = "/var/log" ]; then
-      echo "Preparing $device by migrating logs from $mount_location"
-      mkdir -p /var/log-prep
-      mount $device /var/log-prep
-      mkdir -p /var/log-prep/journal
-      systemctl is-active chronyd > /dev/null && systemctl stop chronyd || :
-      systemctl is-active tuned > /dev/null && systemctl stop tuned || :
-      # rsyslog shouldn't be active but in case it is stop it as well
-      systemctl is-active rsyslog > /dev/null && systemctl stop rsyslog || :
-      cp -a /var/log/. /var/log-prep/
-      umount /var/log-prep
-      rmdir /var/log-prep
-      rm -rf /var/log
-      mkdir -p /var/log
-      echo -n "Mounting: $device to $mount_location"
-      until grep ^$device /etc/mtab > /dev/null; do sleep 1; echo -n .; mount "$mount_location"; done
-      echo
-      systemd-tmpfiles --create --prefix /var/log/journal || :
-      systemctl restart systemd-journald || :
-      systemctl is-enabled tuned > /dev/null && systemctl start tuned || :
-      systemctl is-enabled chronyd > /dev/null && systemctl start chronyd || :
-    else
-      echo -n "Mounting: $device to $mount_location"
-      until grep ^$device /etc/mtab > /dev/null; do sleep 1; echo -n .; mount "$mount_location"; done
-      echo
-    fi
+    echo -n "Mounting: $device to $mount_location"
+    mount "$mount_location"
```

Comment:
Without the until loop mounts will fail sometimes. We had systems where EBS volumes couldn't be mounted directly after being formatted, a couple seconds later the device was available and mountable. That's why the loop was added in the first place.